tests/php/ORM/Filters/EndsWithFilterTest.php

```diff
@@ -267,7 +268,7 @@ public function testMatches(mixed $filterValue, mixed $matchValue, array $modifi
                 }
             }
 
-            EndsWithFilter::config()->set('default_case_sensitive', $caseSensitive);
+            SearchFilter::config()->set('default_case_sensitive', $caseSensitive);
```

Comment:
These were kind of working by accident before. The config in on SearchFilter, not EndsWithFilter

Ditto for the 3x other *MatchFilterTest.php files

Reply:
It wasn't by accident, that was intentional. You can change the config for each filter, fi you want to.